tests/squash.bats

```diff
@@ -132,6 +132,8 @@ function check_lengths() {
   _prefetch alpine
   # populate cache from simple build
   run_buildah build --layers -t test $WITH_POLICY_JSON -f $BUDFILES/layers-squash/Dockerfile.multi-stage
+  run_buildah inspect -t image -f '{{len .Docker.RootFS.DiffIDs}}' test
+  expect_output "1" "should only container 1 diff"
```

Comment:
```suggestion
  expect_output "1" "image should only include 1 layer"
```
Why does this test pass if the image was built with multiple layers, with no squashing requested, producing a final image that contained the layers from its base image, and layers for each of two build steps?